Adaptors/MongoDB/src/TaskTable.cs

```diff
@@ -272,6 +272,10 @@ public async Task<long> UpdateManyTasks(Expression<Func<TaskData, bool>>
                                                       cancellationToken: cancellationToken)
                                      .ConfigureAwait(false);
 
+    Logger.LogInformation("Update tasks matching {condition} and {@updates}",
```

Comment:
I wonder if it would be better to have this in Debug.

Reply:
I think we need to have it in information so that we have all the status update in logs. Actually, that's the log I really want to see.